debian/snapd.autoimport.udev

```diff
@@ -0,0 +1,3 @@
+# probe for assertions, must run before udisks2
+ACTION=="add", SUBSYSTEM=="block" \
+    RUN+="/usr/bin/unshare -m /usr/bin/snap auto-import /dev/%k"
```

Comment:
You can perhaps add `--propagation private` here. It is default but this may change later and it would be very explicit as to what we are doing.